src/terminal/terminal.c

```diff
@@ -1590,17 +1590,120 @@ static int __guac_terminal_send_key(guac_terminal* term, int keysym, int pressed
                 if (keysym == 0xFF54 || keysym == 0xFF99) return guac_terminal_send_string(term, "\x1B[B"); /* Down */
             }
 
+            /* Check if any Modifier key is pressed. */
+            if (term->mod_ctrl || term->mod_alt || term->mod_shift) {
+                int keyseq = 0;
+
+                if (term->mod_ctrl) {
+                    keyseq = keyseq | 4; /* OR operation with other modifier key combination. */
+                }
+
+                if (term->mod_alt) {
+                    keyseq = keyseq | 2; 
+                }
+                
+                if (term->mod_shift) {
+                    keyseq = keyseq | 1;
+                }
+
+                    keyseq = keyseq + 1;
```

Comment:
What's up with the indentation here?

Reply:
Okay so let me fix that
`if (term->mod_alt)        keyseq = keyseq | 2; `
It should be like this right, like in a single line.